tests/integration/ContentTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Transfer\EzPlatform\Tests\Integration;
+
+use Transfer\Adapter\Transaction\Request;
+use Transfer\EzPlatform\Adapter\EzPlatformAdapter;
+use Transfer\EzPlatform\Data\ContentObject;
+use Transfer\EzPlatform\Tests\EzPlatformTestCase;
+
+class ContentTest extends EzPlatformTestCase
+{
+    /**
+     * @var EzPlatformAdapter
+     */
+    protected $adapter;
+
+    public function setUp()
+    {
+        $this->adapter = new EzPlatformAdapter(array(
+            'repository' => static::$repository,
+        ));
+    }
+
+    /**
+     * Tests content creation
+     */
+    public function testCreateContent()
+    {
+        $contentObject = new ContentObject(
+            array(
+                'title' => 'Test title',
+            ),
+            array(
+                'language' => 'eng-GB',
+                'content_type_identifier' => '_test_article',
+                'remote_id' => 'test_article_1'
+            )
+        );
+
+        $this->adapter->send(new Request(array(
+            $contentObject
+        )));
+
+        $content = static::$repository->getContentService()->loadContentByRemoteId('test_article_1');
+
+        $this->assertInstanceOf('\eZ\Publish\API\Repository\Values\Content\Content', $content);
+        $this->assertEquals('Test title', $content->contentInfo->name);
+        $this->assertEquals('Test title', $content->fields['title']['eng-GB']->text);
+        $this->assertEquals('eng-GB', $content->contentInfo->mainLanguageCode);
+        $this->assertEquals(36, $content->contentInfo->contentTypeId);
+    }
+
+    /**
+     * Tests content update
+     */
+    public function testUpdateContent()
+    {
+        $contentObject = new ContentObject(
+            array(
+                'title' => 'Test updated title',
+            ),
+            array(
+                'language' => 'eng-GB',
+                'content_type_identifier' => '_test_article',
+                'remote_id' => 'test_article_1'
+            )
+        );
+
+        $this->adapter->send(new Request(array(
+            $contentObject
+        )));
+
+        $content = static::$repository->getContentService()->loadContentByRemoteId('test_article_1');
+
+        $this->assertInstanceOf('\eZ\Publish\API\Repository\Values\Content\Content', $content);
+        //$this->assertEquals('Test updated title', $content->contentInfo->name);
```

Comment:
No, it's related to `name_schema` of the contenttype. contentInfo->name is not an attribute. Often this equals the first attribute of the ContentType, but it isn't necessarily so. You should use `$content->getField('attributename', '<optional language code>')->value(->text)` instead.